package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "webpack-dev-middleware",
+  "name": "@jonny/webpack-dev-middleware",
```

Comment:
Please fix it

Reply:
Will clean it up and mark the PR as ready!